doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -105,20 +125,52 @@ public void anEmptyCrossRefDocumentThrowsIllegalArgumentException() throws Illeg
         assertThat(exception.getMessage(), is(CrossRefConverter.INVALID_ENTRY_ERROR));
     }
 
+    @Test
+    @DisplayName("An CrossRef document with unknown type throws IllegalArgument exception")
+    public void anCrossRefDocumentWithUnknownTypeThrowsIllegalArgumentException() throws IllegalArgumentException {
+        CrossRefDocument doc = new CrossRefDocument();
+        setAuthor(doc);
+        setPublicationDate(doc);
+        setIssuedDate(doc);    // Issued is required from CrossRef, is either printed-date or
```

Comment:
or ?

Reply:
It is assigned by crossref, as 'Earliest of published-print and published-online'